internal/cli/updog.go

```diff
@@ -0,0 +1,152 @@
+package cli
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/tilt-dev/tilt/internal/analytics"
+	"github.com/tilt-dev/tilt/internal/controllers"
+	"github.com/tilt-dev/tilt/internal/engine"
+	engineanalytics "github.com/tilt-dev/tilt/internal/engine/analytics"
+	"github.com/tilt-dev/tilt/internal/hud"
+	"github.com/tilt-dev/tilt/internal/hud/prompt"
+	"github.com/tilt-dev/tilt/internal/hud/server"
+	"github.com/tilt-dev/tilt/internal/store"
+	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+	"github.com/tilt-dev/tilt/pkg/logger"
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+type updogCmd struct {
+}
+
+var _ tiltCmd = &updogCmd{}
+
+func newUpdogCmd() *updogCmd {
+	return &updogCmd{}
+}
+
+func (c *updogCmd) name() model.TiltSubcommand { return "updog" }
+
+func (c *updogCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "updog",
+		Short: "Runs the Tilt apiserver, applies the config, and streams logs until ctrl-c",
```

Comment:
maybe short doc should mention that this is a dummy app?

Reply:
done!